### ISSUE TITLE
Add Timeline Block to New Report Page

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#241]
## What problem is the feature trying to solve?
With #204, we've identified more useful reporting for accessibility pros. One of the key missing features is a timeline feature that shows the number of alerts that have been marked "equalified" over time. This feature is useful because many governing bodies access accessibility based on the number of fixes you have done over time.

## Desired Solution
On a single report page (#232), the timeline would include a month-by-month chart of the percentage of alerts marked with the status "Equalified". 

Because users can select different blocks to appear on the page, we'll need to make sure the timeline responds to what content is showing.

If either the Grade (#242) or Tag Snapshot (#242) is present, or they are both present, the timeline will show six months of data with the option to browse newer or older months.

If only the timeline is present, the timeline will show 12 months and be positioned full width.

"Next" and "Previous" buttons will allow a user to visit earlier or later blocks of months if there is data for those months.

Users would be able to toggle this block on and off from the report editor page #230.

## Technical Details 
- We may need to consider how this feature queries content, as mentioned in #232. 
- We'll use basic styles from [Bootstrap](http://getbootstrap.com)

## Steps to Completion

- [ ] Develop a "dummy" Twig template, using sample data in the controller.
- [ ] Confirm features via User Testing.
- [ ] Make updates from User Testing lessons.
- [ ] Replace sample data with actual features.
- [ ] Integrate page and related features into the core codebase.

**Comment by bbertucc** [comment]
This was completed in our yet-to-be-released v1. Closing it here to avoid redundancy.
